### PR TITLE
Update trove doc1

### PR DESCRIPTION
**Pull request by ghost**
Adding missing APIs and descriptions to the trove document

### CODE REVIEW

File: learn/trove.md [hunk 1]
```diff
@@ -7,16 +7,56 @@ nav: dbaas
...
+## Database Instance
+
+It is the instantiation of a particular datastore image and version.
+
+**List Database Instances**
+
+To retrieve list of database instances see the example below:
+
+{:.prettyprint .lang-java}
+    List<? extends Instance> list = osv2.trove().instanceService().list();
```
Comment (auhlig) [inline learn/trove.md]: should be `os()` since you refer to the client as `os()`in the other cases ?

File: learn/trove.md [hunk 2]
```diff
@@ -7,16 +7,56 @@ nav: dbaas
...
+
+**Create Database Instance**
+
+{:.prettyprint .lang-java}
+    InstanceCreate instanceCreate = Builders.trove()
+                .instanceCreate().name("testInstance").flavor("1")
+                .volumeSize(6).build();
+    Instance instance = os.trove().instanceService().create(instanceCreate);
```
Comment (auhlig) [inline learn/trove.md]: `os`-> `os()`

File: learn/trove.md [hunk 3]
```diff
@@ -7,16 +7,56 @@ nav: dbaas
...
+
+**Get Database Instance**
+
+To retrieve details of a specific database instance see the example below:
+
+{:.prettyprint .lang-java}
+    TroveInstance troveInstance = (TroveInstance)os.trove().instanceService().get(dbInstanceId);
```
Comment (auhlig) [inline learn/trove.md]: `os` -> `os()`

File: learn/trove.md [hunk 4]
```diff
@@ -7,16 +7,56 @@ nav: dbaas
...
+
+**Delete Database Instance**
+
+To delete a database instance see the example below:
+
+{:.prettyprint .lang-java}
+    ActionResponse actionResponse = os.trove().instanceService().delete(dbInstanceId);
```
Comment (auhlig) [inline learn/trove.md]: `os` -> `os()`

**Review by auhlig — COMMENTED** [review]
Besides the minor things commented this looks good. Many thanks for keeping the docs up to date @sumit74

File: learn/trove.md (hunk 1 above)
Comment (ghost) [reply learn/trove.md]: My Bad. Fixed at all the places.

**Review by auhlig — APPROVED** [review]
Many thanks @sumit74

**Pull request merged**